File: examples/hello.js
#!/usr/bin/env rhino
//.hashdot.profile = shortlived
//.test.prop = "hello world!"
//

msg = java.lang.System.getProperty( "test.prop" )
if( msg == null ) {
    throw "Property 'test.prop' not set"
}

print( msg )

File: examples/hello.clj
#!/usr/bin/env clj
;;.hashdot.profile = shortlived
;;.test.prop = "hello world!"
;;

(let [msg (System/getProperty "test.prop")]
  (if msg
    (println msg)
    (throw (RuntimeException. "Property 'test.prop' not set"))))
